features/backend/countries.feature

```diff
@@ -1,13 +1,13 @@
 @addressing
-Feature: Countries and provinces
+Feature: Countries and administrative areas
```

Comment:
`administrative area`, `locality`, `dependent locality` - these are such French terms! :)  Sounds like CommerceGuys alright.

Perhaps accurate, not particularly clear to me. Locality is OK, the others...